flink-tests/src/test/java/org/apache/flink/test/misc/JobIDLoggingITCase.java

```diff
@@ -205,6 +268,8 @@ private static JobID runJob(ClusterClient<?> clusterClient) throws Exception {
         // wait for all tasks ready and then checkpoint
         while (true) {
             try {
+                clusterClient.triggerCheckpoint(jobId, CheckpointType.DEFAULT).get();
+                // get checkpoint notification
```

Comment:
why do need a second checkpoint trigger to get the checkpoint notification? because the 2nd checkpoint is waiting for the first one to finish? :thinking:

Reply:
I wanted to check the log message when checkpoint completion notification is received. 
That notification is not guaranteed to be sent if the job is cancelled right after the checkpoint, so I added a 2nd checkpoint.

Reply:
:+1: That might be a more descriptive comment than the one in the code right now.